lighthouse-core/report/v2/renderer/report-renderer.js

```diff
@@ -69,17 +88,91 @@ class ReportRenderer {
 
   /**
    * @param {!ReportRenderer.ReportJSON} report
+   * @return {!DocumentFragment}
+   */
+  _renderReportHeader(report) {
+    const header = this._dom.cloneTemplate('#tmpl-lh-heading', this._templateContext);
+    this._dom.find('.lh-config__timestamp', header).textContent =
+        formatDateTime(report.generatedTime);
+    const url = this._dom.find('.lh-metadata__url', header);
+    url.href = report.url;
+    url.textContent = report.url;
+
+    const env = this._dom.find('.lh-env__items', header);
+    report.runtimeConfig.environment.forEach(runtime => {
```

Comment:
s/runtime/env, maybe? (would need to rename `env` above to `envContainer` or whatever)